### DISCUSSION TITLE
我爱绳网，我太tm爱绳网了

### DISCUSSION THREAD

**Discussion by kanmanhin** [share121/inter-knot#5428] [灌水]
绳网是我家，制杖靠大家！我要将绳网，搞得一锅粥！

**Comment by KELERSAMA** [comment]
玛德阿库娅
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >
